components/CanvasOne.js

```diff
@@ -0,0 +1,54 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function CanvasOne({
```

Comment:
Why is the name of the component `CanvasOne`?  Do you have more than one canvas component? 🤔 Usually we prefer to not use numbers for the naming of your components. If a number indicates a type of order of components it reduces the possible reusability of your component. :)